kafka-connect/src/main/java/io/tabular/iceberg/connect/data/DefaultWriteExceptionHandler.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.tabular.iceberg.connect.data;
+
+import io.tabular.iceberg.connect.IcebergSinkConfig;
+import io.tabular.iceberg.connect.deadletter.FailedRecordFactory;
+import org.apache.iceberg.exceptions.ValidationException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.kafka.connect.sink.SinkTaskContext;
+
+public class DefaultWriteExceptionHandler implements WriteExceptionHandler {
```

Comment:
WriteExceptionHandler is only used by the connector, not by both the connector and the error transform --it doesn't need to be shared. 

Since it all gets packaged into a single jar anyways.... I could move it, or leave it separate.  I'm ok with it how it is unless this really bugs you for some reason.  

Note: we have other open comments about modules so let's chat offline about all that.

Reply:
Let me think about this.  If `iceberg-kafka-connect` runtime was `provided`, they should be able to see this no?